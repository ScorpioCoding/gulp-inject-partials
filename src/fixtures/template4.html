<!DOCTYPE html>
<html>
<head>
	<title>gulp-inject</title>
</head>
<body>
	<##:partial1.html>
	</##>
</body>
</html>