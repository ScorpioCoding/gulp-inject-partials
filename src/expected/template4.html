<!DOCTYPE html>
<html>
<head>
	<title>gulp-inject</title>
</head>
<body>
	<##:partial1.html>
	<div>In partial 1</div>
	</##>
</body>
</html>